NICN1.py

```diff
@@ -0,0 +1,283 @@
+from typing import List, Union
+import os
+from Bio import SeqIO
+from Bio import SeqUtils
+from abc import ABC, abstractmethod
```

Comment:
По PEP-8 корректней было бы импорты записать так:
```suggestion
import os
from abc import ABC, abstractmethod
from typing import List, Union

from Bio import SeqIO
from Bio import SeqUtils
```
Также импорты из Bio можно было бы записать в одну строку, но это уже вкусовщина :)
В средах разработки такие моменты можно отслеживать автоматически, рекомендую, очень удобно!